scripts/nipin_comparison.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
```

Comment:
this file is for internal use, not to be packaged in official SCT releases.  i suggest you put it under /dev/nipin

Reply:
Messed up again, wiil fix all those "files that should not be there problem"